### PR TITLE
1.6.2

### PR DESCRIPTION
**Pull request by GhexterCortes**

### CODE REVIEW

**Review by GhexterCortes — COMMENTED** [review]
dam

**Review by GhexterCortes — COMMENTED** [review]
:D

File: index.js [hunk 1]
```diff
@@ -59,170 +59,22 @@ if(config.logging.enabled) {
...
 Client.on('ready', async () => {
     log.warn('Client connected!', 'Status');
     log.warn(`\nInvite: ${ Client.AxisUtility.createInvite(Client) }\n`, 'Invite');
 
-    // Register interaction commands
+    // Load modules
     await Client.AxisUtility.loadModules(config.modulesFolder);
 
-    // On command execution
+    // commands execution
```
Comment (BlobHuman) [inline index.js:73]:
```suggestion
    // Command execution
```

File: modules/axis.js [hunk 2]
```diff
@@ -333,12 +338,12 @@ function makePages(visibleCommands, allCommands, client, language, prefix, embed
...
-        visibleCommands = filterVisibleCommands(visibleCommands, filter, message.member, Client.AxisUtility.get().config.permissions.messageCommands);
+        visibleCommands = filterVisibleCommands(visibleCommands, filter, message.member, Client.AxisUtility.config.permissions.messageCommands);
     
     // Create embeds
-    const embeds = makePages(visibleCommands, commands.MessageCommands, Client, Client.AxisUtility.get().language, Client.AxisUtility.get().config.commandPrefix, Client.AxisUtility.get().config.embedColor);
+    const embeds = makePages(visibleCommands, commands.MessageCommands, Client, Client.AxisUtility.language, Client.AxisUtility.config.commandPrefix, Client.AxisUtility.config.embedColor);
     
-    if(embeds.length == 1) {
+    if(embeds.length <= 1) {
         return SafeMessage.send(message.channel, { embeds: embeds });
```
Comment (BlobHuman) [inline modules/axis.js:347]: idk but
```suggestion
        return SafeMessage.send(message.channel, { content: ' ', embeds: embeds });
```

File: modules/axis.js [hunk 3]
```diff
@@ -347,14 +352,14 @@ async function getHelpMessage(args, message, Client) {
...
     
     // Create embeds
-    const embeds = makePages(visibleCommands, commands.InteractionCommands, Client, Client.AxisUtility.get().language, '/', Client.AxisUtility.get().config.embedColor);
+    const embeds = makePages(visibleCommands, commands.InteractionCommands, Client, Client.AxisUtility.language, '/', Client.AxisUtility.config.embedColor);
 
     // Send response
     await SafeInteract.deferReply(interaction);
-    if(embeds.length == 1) { 
+    if(embeds.length <= 1) { 
         return SafeInteract.editReply(interaction, { embeds: embeds });
```
Comment (BlobHuman) [inline modules/axis.js:363]:
```suggestion
        return SafeInteract.editReply(interaction, { content: ' ', embeds: embeds });
```

File: scripts/Utility.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        if(!options.language) throw new TypeError('No language provided');
+        if(!options.Client) throw new TypeError('No client provided');
+
+        this.logger = options.logger;
+        this.config = options.config;
+        this.language = options.language;
+        this.scripts = {};
+        this.commands = { MessageCommands: [], InteractionCommands: [] };
+        this.Client = options.Client;
+    }
```
Comment (BlobHuman) [inline scripts/Utility.js:30]: Allow setting the commands and scripts optionally
```suggestion
    constructor(options = { logger: new Util.Logger('AxisUtility'), config: null, language: null, Client: null, scripts: [], commands: { MessageCommands: [], InteractionCommands: [] } }) {
        if(!options.config) throw new TypeError('No config provided');
        if(!options.language) throw new TypeError('No language provided');
        if(!options.Client) throw new TypeError('No client provided');

        this.logger = options.logger;
        this.config = options.config;
        this.language = options.language;
        this.scripts = options.scripts?.length ? options.scripts : [];
        this.commands = options.commands ? options.commands : { MessageCommands: [], InteractionCommands: [] };
        this.Client = options.Client;
    }
```

**Review by BlobHuman — CHANGES_REQUESTED** [review]

**Pull request merged**